setup.cfg

```diff
@@ -34,6 +34,8 @@ install_requires =
     scipy
     particletools
     particle
+    tqdm
+    requests
```

Comment:
It looks like you also changed the implementation to use the Python std lib (thanks!), so can we also remove requests from the install requirements?

Reply:
Yes. It is removed.